assemblies/working-with-distributed-workloads.adoc

```diff
@@ -0,0 +1,41 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+
+:context: distributed-workloads
+
+[id="working-with-distributed-workloads_{context}"]
+= Working with distributed workloads
+
+[role='_abstract']
+Distributed workloads enable data scientists to use multiple nodes in parallel to train machine-learning models or process data more quickly.
```

Comment:
Avoid the use of _enable_ here? :)

Reply:
When he refer to _nodes_, I think we're referring to OpenShift worker nodes? If so, I think that's probably worth saying.